editor/js/Editor.js

```diff
@@ -84,14 +85,17 @@ function Editor() {
 		refreshSidebarObject3D: new Signal(),
 		historyChanged: new Signal(),
 
-		viewportCameraChanged: new Signal()
+		viewportCameraChanged: new Signal(),
+
+		intersects: new Signal(),
```

Comment:
What do you think about renaming the signal to `intersectionsDetected`? This name seems to more compliant compared to others. And it feels more descriptive.

Reply:
> What do you think about renaming the signal to `intersectionsDetected`? This name seems to more compliant compared to others. And it feels more descriptive.

@Mugen87 Done. Thank you for your reply. Renaming the signal to `intersectionsDetected` is better. TBH, I am not good at English...